tests/conftest.py

```diff
@@ -20,3 +22,29 @@ def remove_data_folder():
             shutil.rmtree(data)
 
     request.addfinalizer(remove_data_folder)
+
+
+@pytest.fixture
+def tas_series_1d():
+    """Create a temperature time series."""
+
+    def _tas_series(values, start="7/1/2000", units="K", freq="D", as_dataset=True):
+        coords = pd.date_range(start, periods=len(values), freq=freq)
+        da = xr.DataArray(
+            values,
+            coords=[coords],
+            dims="time",
+            name="tas",
+            attrs={
+                "standard_name": "air_temperature",
+                "cell_methods": "time: mean within days",
+                "units": units,
+            },
+        )
+        da.name = "tas"
+        if as_dataset:
+            return da.to_dataset()
+        else:
+            return da
+
+    return _tas_series
```

Comment:
I think we should explore the xclim-based solution. Would you want me and/or @aulemahal to add something there first?

Reply:
Yeah, this is just what I had at the time of our discussion, and helps to show the modifications I had to do (such as expose `freq`). We should indeed work on modifying `xclim` first.

Reply:
@Zeitsperre @aulemahal 
This new commit will fail because `test_timeseries` isn't yet accessible outside of `xclim`'s master branch, but is this closer to what we had in mind ? Or did we want to remove fixtures altogether ?